.travis.yml

```diff
@@ -46,7 +46,13 @@ install:
     fi
   - |
     if [ "$LINT" = "1" ] || [ "$PHP" = "1" ] || [ "$DEPLOY_PLUGIN" = "1" ]; then
-      composer install
+      composer install -n --ignore-platform-reqs
+    fi
+  - |
+    if [ $(phpenv version-name) = "5.6" ] || [ $(phpenv version-name) = "7.0" ]; then
+      # phpstan only supports PHP 7.1+. Ignore version conflicts on lower versions.
+      composer remove -n --dev szepeviktor/phpstan-wordpress --ignore-platform-reqs
+      composer update -n --optimize-autoloader
```

Comment:
Why not add `--no-interaction --no-suggest` here?

Reply:
`-n` is `--no-interaction` - although not nice to be short

Reply:
Why not nice? Not clear what it does?

Reply:
If someone sees `--no-interaction` and speaks a little bit of English he will know what it is for, `-n` is not like that!